lua/pulsar_lib/core/sql/sv_sql.lua

```diff
@@ -92,8 +127,63 @@ function SQL:RawQuery(query, onSuccess, onError)
 
 		local x = string.Split(query, "\n")
 		for _, line in ipairs(x) do
-			PulsarStore.Logging.Debug(line)
+			PulsarLib.Logging.Debug(line)
+		end
+		query = sql.Query(query)
+
+		if query == false then
+			PulsarLib.Logging.Fatal("SQL query failed!")
+			PulsarLib.Logging.Fatal(sql.LastError())
+			onError(sql.LastError())
+		else
+			onSuccess(query)
+		end
+	end
+end
+
+function SQL:PreparedQuery(query, values, onSuccess, onError)
+	onSuccess = onSuccess or emptyFunction
+	onError = onError or emptyFunction
+
+	if self.Details.UsingMySQL then
+		local queryObj = self.Connection:prepare(query)
+
+		queryObj.onSuccess = function(_, data)
+			onSuccess(data)
+		end
+
+		queryObj.onError = function(_, err)
+			PulsarLib.Logging.Fatal("Prepared MySQL query failed!")
+			PulsarLib.Logging.Fatal(err)
+			PulsarLib.Logging.Fatal(SQL:prepareStatement(query, values))
```

Comment:
Can these calls be replaced with `local logger = PulsarLib.Logging:Get("Database")` -> `logger.Fatal(whatever)` please?